src/Employer/Employer.Web/Orchestrators/Part1/EmployerOrchestrator.cs

```diff
@@ -20,18 +19,19 @@ public EmployerOrchestrator(IVacancyClient client)
             _client = client;
         }
 
-        public async Task<EmployerViewModel> GetLocationViewModelAsync(Guid vacancyId)
+        public async Task<EmployerViewModel> GetEmployerViewModelAsync(VacancyRouteModel vrm)
         {
-            var vacancy = await _client.GetVacancyForEditAsync(vacancyId);
-            
+            var getEmployerDataTask = _client.GetEmployerVacancyDataAsync(vrm.EmployerAccountId);
+            var getVacancyTask = _client.GetVacancyForEditAsync(vrm.VacancyId);
+
+            await Task.WhenAll(new Task[] { getEmployerDataTask, getVacancyTask });
+
+            var employerData = getEmployerDataTask.Result;
+            var vacancy = getVacancyTask.Result;
+
             var vm = new EmployerViewModel
             {
-                Organisations = new List<LocationOrganisationViewModel>
-                {
-                    new LocationOrganisationViewModel{Id = "A", Name = "Organisation A"},
-                    new LocationOrganisationViewModel{Id = "B", Name = "Organisation B"},
-                    new LocationOrganisationViewModel{Id = "C", Name = "Organisation C"}
-                },
+                Organisations = employerData.LegalEntities.Select(MapLegalEntitiesToOrgs).ToList(),
```

Comment:
Subjective - but I'd prefer an extension method here. ` employerData.LegalEntities.ToViewModel()`

Reply:
We'll be making changes to this shortly.